ArtOfIllusion/src/artofillusion/ScrollViewTool.java

```diff
@@ -0,0 +1,301 @@
+/* Copyright (C) 2017 by Petri Ihalainen
```

Comment:
This file concerns me a bit. Its not really a tool at all, its a scrollEvent handler. 

The existing MoveViewTool and RotateViewTool are real, clickable tools that are also set up to be activated by <meta>(<ctrl>) and <alt> keys. The way that this experiment is developing, I'm starting to think that we should consider reverting the two to their 'classic' style of operation, and leaving them as click-only tools. 

The more inclusive movement controls that you have been working on should be handled as MouseEvent handlers in the canvas itself, or as an inclusive specialized mouse-handler class. In either case, it should be separated from the 'ViewXYZTools' as such. I don't think we want another 'tool' that does not really fit in the tool paradigm.